lib/rules/web/admin_console/4.5/clustersettings.xyaml

```diff
@@ -0,0 +1,26 @@
+click_update_channel_button:
+  elements:
+  - selector:
+      xpath: //button[@data-test-id='current-channel-update-link']
+    op: click
+click_channels_drop_down:
+  elements:
+  - selector:
+      xpath: //button[@data-test-id='dropdown-button']
+    op: click
+check_drop_down_items:
+  elements:
+  - selector:
+      xpath: //button[@data-test-id='dropdown-menu' and contains(., '<menu_item_1>')]
```

Comment:
It looks like we don't need `params`
```
//button[@data-test-id='dropdown-menu' and contains(., 'stable-<product_version>')]
```

Reply:
Also on OCP 4.5, we also need to check there are `stable-4.6, fast-4.6, candidate-4.6` channels as well

Reply:
@yapei Updates:
- removed the params
- updated the xpath to have the params value instead.
- updated the corresponding version xyaml files 
     note the drop down values have additional version in 4.4 only. Both 4.5 and 4.6 have just one set of values.

Reply:
@schituku 

>  note the drop down values have additional version in 4.4 only. Both 4.5 and 4.6 have just one set of values.

Yes, after 4.6 is released we will add `stable-4.6`, `fast-4.6` and `candidate-4.6` channels to 4.5 cluster. And when 4.7 is released we will add `stable-4.7`, `fast-4.7` and `candidate-4.7` channels to 4.6 cluster.